server/src/routes/comment.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, Router } from "express";
+
+import { createComment, getAllComments } from "../controller/comment";
+
+const router: Router = Router();
+
+router.post("/", async (req: Request, res: Response) => {
```

Comment:
You should validate the data using a validator

Reply:
You should validate the data using a validator

router.post("/", validateData(commentValidator), async(req: Request, res: Response))